src/python/test/test_dxclient.py

```diff
@@ -2770,98 +2770,104 @@ def _get_analysis_id(dx_run_output):
         # --watch implies high priority when --priority is not specified
         try:
```

Comment:
Can you wrap most of this test with `with self.configure_ssh() as wd:` to set up the keypair and update the public key of the test user?

Reply:
https://jenkins-vstg.internal.dnanexus.com/job/dx-toolkit/job/dxpy-branch-integration-tests-on-staging-20.04-python3/77/console
I have wrapped those tests. Works great! Thanks! @kpjensen